test/jdk/java/io/Console/ConsolePromptTest.java

```diff
@@ -57,34 +71,35 @@ void testCorrectOutputReadPassword() throws Exception {
     void doRunConsoleTest(String testName,
                           String input,
                           String expectedOut) throws Exception {
-        ProcessBuilder builder =
-                ProcessTools.createTestJavaProcessBuilder(ConsoleTest.class.getName(),
-                                                          testName);
-        OutputAnalyzer output = ProcessTools.executeProcess(builder, input);
-
-        output.waitFor();
-
-        if (output.getExitValue() != 0) {
-            throw new AssertionError("Unexpected return value: " + output.getExitValue() +
-                                     ", actualOut: " + output.getStdout() +
-                                     ", actualErr: " + output.getStderr());
+        // check "expect" command availability
+        var expect = Paths.get("/usr/bin/expect");
+        if (!Files.exists(expect) || !Files.isExecutable(expect)) {
+            System.out.println("'expect' command not found. Test ignored.");
```

Comment:
Hello Naoto, I think throwing a `jtreg.SkippedException` might be better here so that it's clear that the test was skipped. There have been recent reporting improvements too which make it easier to notice such skipped tests.

Reply:
Good point. Modified the piece (w/ some other minor changes)